testing/regress/ecl/parquetTypes.ecl

```diff
@@ -335,16 +313,45 @@ END;
 unicodeCompareResult := JOIN(unicodeDatasetOut, unicodeDatasetIn, LEFT.testid = RIGHT.testid, unicodeJoin(LEFT, RIGHT), ALL);
 unicodeResult := IF(COUNT(unicodeCompareResult(isEqual = FALSE)) = 0, 'Pass', 'Fail');
 
-PARALLEL(
-    OUTPUT(booleanResult, NAMED('BooleanTest'), OVERWRITE),
-    OUTPUT(integerResult, NAMED('IntegerTest'), OVERWRITE),
-    OUTPUT(unsignedResult, NAMED('UnsignedTest'), OVERWRITE),
-    OUTPUT(realResult, NAMED('RealTest'), OVERWRITE),
-    OUTPUT(decimalResult, NAMED('DecimalTest'), OVERWRITE),
-    OUTPUT(stringResult, NAMED('StringTest'), OVERWRITE),
-    OUTPUT(dataResult, NAMED('DataAsStringTest'), OVERWRITE),
-    OUTPUT(varStringResult, NAMED('VarStringTest'), OVERWRITE),
-    OUTPUT(qStringResult, NAMED('QStringTest'), OVERWRITE),
-    OUTPUT(utf8Result, NAMED('UTF8Test'), OVERWRITE),
-    OUTPUT(unicodeResult, NAMED('UnicodeTest'), OVERWRITE)
+SEQUENTIAL(
+    // Set up test files
+    PARALLEL(
+        ParquetIO.Write(booleanDatasetOut, dropzoneDirectory + 'BooleanTest.parquet', TRUE, compressionType),
+        ParquetIO.Write(integerDatasetOut, dropzoneDirectory + 'IntegerTest.parquet', TRUE, compressionType),
+        ParquetIO.Write(unsignedDatasetOut, dropzoneDirectory + 'UnsignedTest.parquet', TRUE, compressionType),
+        ParquetIO.Write(realDatasetOut, dropzoneDirectory + 'RealTest.parquet', TRUE, compressionType),
+        ParquetIO.Write(decimalDatasetOut, dropzoneDirectory + 'DecimalTest.parquet', TRUE, compressionType),
+        ParquetIO.Write(stringDatasetOut, dropzoneDirectory + 'StringTest.parquet', TRUE, compressionType),
+        ParquetIO.Write(dataDatasetOut, dropzoneDirectory + 'DataTest.parquet', TRUE, compressionType),
+        ParquetIO.Write(varStringDatasetOut, dropzoneDirectory + 'VarStringTest.parquet', TRUE, compressionType),
+        ParquetIO.Write(qStringDatasetOut, dropzoneDirectory + 'QStringTest.parquet', TRUE, compressionType),
+        ParquetIO.Write(utf8DatasetOut, dropzoneDirectory + 'UTF8Test.parquet', TRUE, compressionType),
+        ParquetIO.Write(unicodeDatasetOut, dropzoneDirectory + 'UnicodeTest.parquet', TRUE, compressionType),
+    ),
+    // Read and compare results
+    OUTPUT(booleanResult, NAMED('BooleanTest')),
```

Comment:
Do you want all of these in a nested PARALLEL block as well?  As it stands nothing will be commoned up between them.  That may not matter.

Reply:
ok.  No need to change for this PR.  For future reference, ORDERED(a,b,c) preserves the order the items are output, but allows code to be commoned up between them.